Containers/nextcloud/Dockerfile

```diff
@@ -205,6 +205,7 @@ RUN set -ex; \
         mawk \
         sudo \
         grep \
+        nodejs \
```

Comment:
Does it require `nodejs ` (lts) or `nodejs-current` (current)?

Reply:
cc @marcelklehr?

Reply:
I think nodejs should be sufficient